src/model.jl

```diff
@@ -51,16 +55,16 @@ function PowerManagementProblem(fq, fl, d, pmax, gmax, A, B; τ=TAU, ds=0)
         + (τ/2)*sumsquares(p)
     )
     add_constraints!(problem, [
-        -p <= pmax,
-        p <= pmax,
-        -g <= 0, 
-        g <= gmax,
-        0 == A*p - B*g + d + ds,
+        -p <= pmax, #λpl
+        p <= pmax, #λpu
+        -g <= 0, #λgl
+        g <= gmax, #λgu
+        0 == A*p - B*g + d + ch/η_c - η_d * dis, #ν
```

Comment:
What if we leave `\eta_c` and `\eta_d` out of the subproblems and set `s[t+1] = s[t] + \eta_c * ch[t] - \eta_d * dis[t]` in the `DynamicPowerManagementProblem`? This way, charge and discharge are the power leaving / entering the market, respectively, and the battery only gains `\eta_c * ch` units of energy (or loses `\eta_d * dis` units of energy, respectively).

Reply:
I wrote it the current way because it made charge/discharge power constraints read `ch[t] <= P` for instance, which I find clear and succinct. 

Now, if you want, we can always change that later.

Reply:
We just have to be clear on what the max (dis)charge power constraints for storage actually mean then